cypress/fixtures/performanceModule.json

```diff
@@ -0,0 +1,12 @@
+
+
+    {
+        "localAppURL": "http://localhost:8882/?page_id=2",
+        "localAppURLHover": "http://localhost:8882/",
+        "statusCode": 200,
+        "mouseDownToBeSelected": "Prefetch on Mouse Down",
+        "mouseHoverToBeSelected": "Prefetch on Mouse Hover (Recommended)",
```

Comment:
Since text can vary across languages, do you think it would be better to use IDs, class names, or data attributes instead?